dev/ssl.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+tmp_cert="$HOME/tmp/root_ca.crt"
```

Comment:
may be need to create a `tmp` directory first.
```suggestion
mkdir $HOME/tmp
tmp_cert="$HOME/tmp/root_ca.crt"
```